build-plugins/src/main/kotlin/aws/sdk/kotlin/gradle/kmp/IdeUtils.kt

```diff
@@ -10,13 +10,14 @@ import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 import org.jetbrains.kotlin.konan.target.HostManager
 import org.jetbrains.kotlin.konan.target.KonanTarget
+import java.util.*
 
 /**
  * Whether Intellij is active or not
  */
 val IDEA_ACTIVE = System.getProperty("idea.active") == "true"
 
-val OS_NAME = System.getProperty("os.name").toLowerCase()
+val OS_NAME = System.getProperty("os.name").lowercase(Locale.getDefault())
```

Comment:
**Nit**: Specifying a locale is optional and can be omitted.

Reply:
IDE auto correct